tests/eas/preliminary.py

```diff
@@ -190,41 +190,36 @@ def test_energy_model_present(self):
                 '- No energy model in kernel')
 
 class TestSchedutilTunables(BasicCheckTest):
-    MAX_UP_RATE_LIMIT_US = 20 * 1e3
-    MAX_DOWN_RATE_LIMIT_US = 20 * 1e3
+    MAX_RATE_LIMIT_US = 20 * 1e3
 
     def test_rate_limit_not_too_high(self):
         """Test that the schedutil ratelimiting is not too harsh"""
         governors = self.target.cpufreq.list_governors(0)
         if 'schedutil' not in governors:
             raise SkipTest('schedutil not present on target')
         self.target.cpufreq.set_all_governors('schedutil')
+        tunables  = self.target.cpufreq.list_governor_tunables(0)
```

Comment:
Why is this being set here when `tunables = self.target.cpufreq.get_governor_tunables(cpu)` is in the loop below ?